### PR TITLE
feat: add service-proxy for proxy TCP protocol

### PR DESCRIPTION
**Pull request by bobsongplus**
#### What type of PR is this?

/kind feature

#### What this PR does / why we need it
增加 service-proxy 代理helm chart 

#### Which issue(s) this PR fixes

Fixes #

### CODE REVIEW

**Comment by bobsongplus** [comment]
@nkwangleiGIT

File: charts/cluster-component/charts/service-proxy/values.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  repository: harbor.172.22.96.178.nip.io/system_containers/tenx-proxy
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: "v5.6.0-20231019"
+
+exporterImage:
+  repository: harbor.172.22.96.178.nip.io/system_containers/haproxy-exporter
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: "v0.13.0"
+
+ingress:
```
Comment (nkwangleiGIT) [inline charts/cluster-component/charts/service-proxy/values.yaml]: 这个ingress 有用吗？

Comment (bobsongplus) [reply charts/cluster-component/charts/service-proxy/values.yaml]: 目前没有用到，先删掉？

Comment (nkwangleiGIT) [reply charts/cluster-component/charts/service-proxy/values.yaml]: 嗯，没用就先去掉吧

Comment (bobsongplus) [reply charts/cluster-component/charts/service-proxy/values.yaml]: done

**Pull request merged**